### PR TITLE
Feature/141/privacy policy and acceptance of terms

### PR DESCRIPTION
**Pull request by gluhovSemen**

### CODE REVIEW

File: src/apps/content/templates/support/terms_and_conditions.html [hunk 1]
```diff
@@ -1,21 +1,23 @@
 {% extends "base.html" %}
+{% load static %}
```
Comment (volodin28) [inline src/apps/content/templates/support/terms_and_conditions.html:2]: isn't it better to include {% load static %} to the base.html
so we wouldn't add this line in every other template

Comment (gluhovSemen) [reply src/apps/content/templates/support/terms_and_conditions.html:2]: i think we need static files not so often inside the content block, what other use cases are you thinking off?

Comment (gluhovSemen) [reply src/apps/content/templates/support/terms_and_conditions.html:2]: fixed

File: src/config/settings.py [hunk 2]
```diff
@@ -142,7 +142,7 @@
 STATIC_URL = "static/"
 STATIC_ROOT = BASE_DIR / "static"
 
-STATICFILES_DIRS = [BASE_DIR / "static/bootstrap"]
+STATICFILES_DIRS = [BASE_DIR / "static/bootstrap", "src/static/pdfs"]
```
Comment (egorvavilov) [inline src/config/settings.py]: src/static/pdfs? lets point it to base dir

Comment (gluhovSemen) [reply src/config/settings.py]: It doesnt let me do that, throwing The STATICFILES_DIRS setting should not contain the STATIC_ROOT setting.

Comment (gluhovSemen) [reply src/config/settings.py]: i can do it like this though STATICFILES_DIRS = [BASE_DIR / "static/bootstrap", BASE_DIR/"static/pdfs"], does it make more seance?

**Review by egorvavilov — APPROVED** [review]
lgtm

**Pull request merged**